src/error.rs

```diff
@@ -312,6 +312,13 @@ impl<W> IntoInnerError<W> {
         &self.err
     }
 
+    /// Consumes the [`IntoInnerError`] and returns the error which caused the call to
+    /// [`Writer::into_inner()`] to fail. Unlike `error`, this can be used to
+    /// obtain ownership of the underlying error.
```

Comment:
Please make sure the docs are wrapped to 79 columns, inclusive. (They don't currently appear to be.)

Also, please use `Writer::into_inner` and drop the parens.

And please use `IntoInnerError::error` instead of `error`.

Reply:
Docs wrapped, and I managed to link to `Writer::into_iter()` correctly this time.